crates/brix_errors/src/lib.rs

```diff
@@ -55,6 +98,10 @@ impl Display for BrixErrorKind {
 
 impl Display for BrixError {
     fn fmt(&self, fmt: &mut fmt::Formatter) -> Result<(), fmt::Error> {
-        write!(fmt, "{} error: {}", self.kind, self.message)
+        if let Some(kind) = &self.kind {
+            write!(fmt, "{} error: {}", kind, self.message)
```

Comment:
wouldn't it be a better idea to use a logging facility like I set up to do? or would this still go through that, also I feel like you need more logging than just errors. I was definitely using debug levels when trying to figure out what brix java was doing in some cases.

Reply:
Isn't the logging sort of irrelevant to errors since the output of this fmt function will be what the user sees instead of logging output? Or am I wrong...

Reply:
I'm not sure in rust, in java I simply configured the log output to be at error level at runtime and reduced the formatting output to only be things I cared about.